ledger/ledger-utxo/src/test/java/net/corda/v5/ledger/utxo/uniqueness/client/StateRefParseTest.java

```diff
@@ -0,0 +1,75 @@
+package net.corda.v5.ledger.utxo.uniqueness.client;
```

Comment:
Why is this test in the `...uniqueness.client` package?

Reply:
should this be inside `...uniqueness` rather than `...uniqueness.client`?
I was just trying to match with `StateRef.java` package name

Reply:
Yeah. The same package should be fine. (`net.corda.v5.ledger.utxo`)